internal/store/store.go

```diff
@@ -269,8 +269,22 @@ func WithContext(ctx context.Context) Option {
 	}
 }
 
+// Appends user agent given a build context for telemetry.
+func appendUserAgent(buildContext string) {
+	if !strings.Contains(config.UserAgent, buildContext) {
+		config.UserAgent = fmt.Sprintf("%s-%s", config.UserAgent, buildContext)
```

Comment:
Since there's a space rather than a dash in the existing pattern, I think we should consider doing the same here to be consistent (left a comment on the ticket also)
https://github.com/mongodb/mongodb-atlas-cli/blob/master/internal/config/profile.go#L76

![image](https://github.com/mongodb/mongodb-atlas-cli/assets/10655290/61ab38a4-f5f4-4d45-b7be-211125ef027f)

Reply:
Addressed